client/src/components/DeleteButton.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import deleteIconSrc from "../assets/delete.svg";
+import styled from "@emotion/styled";
+
+const DeleteButton = () => {
+  return (
+    <ButtonContainer>
+      <img src={deleteIconSrc} alt="Dustbin" />
+      <p>To-Do löschen</p>
+    </ButtonContainer>
+  );
+};
+export default DeleteButton;
+
+const ButtonContainer = styled.button`
+  display: flex;
```

Comment:
It's a bit strange, that a button container is a button? normally a container is something like a div or span and the buttons are in this?